### PR TITLE
Feature/pelagos 3862 make method field required

### PR DESCRIPTION
**Pull request by fightingtexasaggie**

### CODE REVIEW

File: src/Form/DatasetSubmissionType.php [hunk 1]
```diff
@@ -132,7 +132,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ))
             ->add('suppMethods', Type\TextareaType::class, array(
                 'label' => 'Supplemental Information - Methods',
-                'required' => false,
+                'required' => true,
```
Comment (praneethpr) [inline src/Form/DatasetSubmissionType.php]: We use the same Form type in Dataset-review tool, and the PO's don't want it to be required in the Dataset-review tool

**Comment by fightingtexasaggie** [comment]
Separating the datasetSubmission and datasetSubmissionReview types creates a heck of a lot of code duplication, but neither inherit_data, php traits, nor inheritance seems to be able to help.  I'm open to ideas.

File: src/Form/DatasetSubmissionType.php (hunk 1 above)
Comment (fightingtexasaggie) [reply src/Form/DatasetSubmissionType.php]: I've separated them now, but if you have a suggestion for code-reuse, let me know. It's a painful amount of duplication.

File: src/Entity/DatasetSubmission.php [hunk 2]
```diff
@@ -334,7 +334,7 @@ class DatasetSubmission extends Entity
             'description' => 'energy, water and waste systems and communications infrastructure and services. Examples: hydroelectricity, geothermal, solar and nuclear sources of energy, water purification and distribution, sewage collection and disposal, electricity and gas distribution, data communication, telecommunication, radio, communication networks',
         ],
     ];
-    
+
```
Comment (praneethpr) [inline src/Entity/DatasetSubmission.php]: Lot of whitespaces on this file

**Pull request merged**